sprout/appliances/tasks.py

```diff
@@ -1203,9 +1203,6 @@ def appliance_power_on(self, appliance_id):
                 # VM is running now.
                 sync_appliance_hw.delay(appliance.id)
                 sync_provider_hw.delay(appliance.template.provider.id)
-                # fixes time synchronization
```

Comment:
I don't think that this needs to or should happen during power on - an unconfigured appliance will have the default NTP servers for the image produced by CFME (rhel NTP pool).  A configured appliance will have either the EVM default (public ntp pools) or the NTP servers set during configuration.